### DISCUSSION TITLE
make build does not reflect changes to aw_client

### DISCUSSION THREAD

**Discussion by j0eW** [ActivityWatch/activitywatch#680] [Q&A] [answered]
Hello,

i just want to make some minor changes to aw_client/client.py but after make build i still have the old version in my venv directory. I also tried with make package but also the old version.

everything else is working as expected.

thanks

**Comment by ErikBjare** [comment, marked as answer]
Sometimes `make build` does something weird when it's first installing a version of aw-client from PyPI and then the one from the folder. You could run `pip show aw-client` to see which folder it's installed in, then run `pip uninstall aw-client`. 

Now, since there may be files left behind (I'm not sure how/why this happens): go into the `site-packages` folder (shown by `pip show aw-client`) and remove all aw-client related files & folders, then install aw-client. It should now work.
